python-sdk/tests/benchmark/dags/evaluate_load_file.py

```diff
@@ -1,21 +1,26 @@
 import json
 import os
 from datetime import datetime
-from pathlib import Path
 
 from airflow import DAG
 from astro import sql as aql
 from astro.constants import DEFAULT_CHUNK_SIZE, FileType
 from astro.files import File
 from astro.sql.table import Metadata, Table
+from google.cloud import storage
+from tests.benchmark import settings as benchmark_settings
 
 START_DATE = datetime(2000, 1, 1)
 
 
 def load_config():
-    config_path = Path(Path(__file__).parent.parent, "config.json").resolve()
-    with open(config_path) as fp:
-        return json.load(fp)
+    tmp_config = "/tmp/config.json"
```

Comment:
Please use [gettempdir](https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir).

Reply:
The `tempfile.NamedTemporaryFile` may be more suitable, if we use something like:
```
from tempfile import NamedTemporaryFile

with tempfile.NamedTemporaryFile(mode="w") as temp_file:
    print(temp_file.name)
```
https://docs.python.org/3/library/tempfile.html

Reply:
Yes, or https://github.com/astronomer/astro-sdk/pull/954#discussion_r983117319